.dockerignore

```diff
@@ -128,6 +131,8 @@ blocks/**/.env
 # generated files
 *.gen.*
 *.tsbuildinfo
+*.pdb
+**/*.rs.bk
```

Comment:
Previously generated by `rustfmt` but we format in place, not needed.

Reply:
I'll remove; ty.

Reply:
Addressed in https://github.com/hashintel/hash/pull/2938/commits/211e32f469a1e5fe75e195250422ebd111c781af